src/relay/index.ts

```diff
@@ -0,0 +1,49 @@
+import { multiaddr } from '@multiformats/multiaddr'
+import { expect } from 'aegir/chai'
+import type { Daemon, DaemonFactory, NodeType, SpawnOptions } from '../index.js'
+import { Status } from './pb/index.js'
+import { reserve } from './util.js'
+
+export function relayTests (factory: DaemonFactory) {
+  const t: NodeType[] = ['go', 'js']
+  t.forEach(a => t.forEach(b => t.forEach(r => { if (a !== b || a !== r) relayTest(factory, a, b, r) })))
```

Comment:
That makes sense. This ensures we skip cases where both implementations are the same. Is that worth skipping? I made the assumption since cases like that should be covered by the implementation's own testing.

Reply:
Both versions are the same, no? I think the second version is easier to understand

https://runkit.com/marcopolo/demo-1234

Reply:
Yes, they are the same. My question was more around whether it's okay to skip those kinds of tests.

Reply:
I think maybe don't skip js -> js and go -> go - it gives a reasonably useful finger-in-the-air-style metric for performance between implementations.